src/components/Breadcrumbs/Breadcrumbs.module.css

```diff
@@ -0,0 +1,35 @@
+.list {
+	padding: 20px 20px;
+	margin: 0;
+	display: flex;
+	list-style: none;
+}
+
+.list li {
```

Comment:
Not too good to use specific selectors (like `li` or `li button`) because in your `jsx` you can change the element to `div` for example, but will forget to change it here.